pandora/client.py

```diff
@@ -227,3 +248,20 @@ def share_music(self, music_token, *emails):
         return self("music.shareMusic",
                     musicToken=music_token,
                     email=emails[0])
+
+    def get_ad_item(self, ad_token):
+        from .models.pandora import AdItem
+
+        return AdItem.from_json(self, self.get_ad_metadata(ad_token))
```

Comment:
The convention of the client is to mix is to mix the low level call with the model building when returning rich data models. I don't see `get_ad_metadata` called anywhere else, can you combine these two methods?